demisto_sdk/commands/common/logger.py

```diff
@@ -2,28 +2,85 @@
 import logging
 import logging.config
 import os.path
+import platform
 import re
 import sys
 from logging.handlers import RotatingFileHandler
 from pathlib import Path
 from typing import Dict, List, Optional, Union
 
-from demisto_sdk.commands.common.content_constant_paths import CONTENT_PATH
-from demisto_sdk.commands.common.tools import parse_int_or_default, string_to_bool
+# NOTE: Do not add internal imports here, as it may cause circular imports.
+from demisto_sdk.commands.common.constants import STRING_TO_BOOL_MAP
 
 logger: logging.Logger = logging.getLogger("demisto-sdk")
 
+
+def environment_variable_to_bool(
+    variable_name: str, default_value: bool = False
+) -> bool:
+    """
+    Check if the environment variable is set and is a valid boolean value.
+    If it is not set or is not a valid boolean value, return the default value.
+
+    Args:
+        variable_name (str): The name of the environment variable.
+        default_value (bool): A default value to return if the environment variable is not set or is invalid.
+
+    Returns:
+        bool: The environment variable value if it is set and is a valid boolean value, otherwise the default value.
+    """
+    env_var = os.getenv(variable_name, default_value)
+
+    if isinstance(env_var, bool):
+        return env_var
+
+    return (
+        isinstance(env_var, str)
+        and env_var.casefold() in STRING_TO_BOOL_MAP
+        and STRING_TO_BOOL_MAP[env_var]
+    )
+
+
+def environment_variable_to_int(variable_name: str, default_value: int) -> int:
```

Comment:
should we force to send here a default value?

Reply:
This matches the current uses for these functions (keeping their functionality as they were before).
I can update it, but it might add unneeded complexity (and unused logic).

If no default value is provided, and the environment variable is missing, what should the behavior be? Raise an error?